lib/src/svg/xml_parsers.dart

```diff
@@ -125,19 +125,19 @@ PaintServer parseLinearGradient(XmlElement el) {
 
   parseStops(stops, colors, offsets);
 
+  final Matrix4 transform = parseTransform(getAttribute(el, 'gradientTransform', def: null));
+  
   return (Rect bounds) {
-    final Offset from = new Offset(
-      bounds.left + (bounds.width * x1),
-      bounds.left + (bounds.height * y1),
-    );
-    final Offset to = new Offset(
-      bounds.left + (bounds.width * x2),
-      bounds.left + (bounds.height * y2),
-    );
+    final from = Vector3(bounds.left + x1, bounds.top + y1, 0.0);
+    final to = Vector3(bounds.left + x2, bounds.top + y2, 0.0);
+    if(transform != null){
+      transform.transform3(from);
+      transform.transform3(to);
+    }
 
     final Gradient gradient = new Gradient.linear(
-      from,
-      to,
+      Offset(from.x, from.y),
```

Comment:
I haven't switched things over to avoid using `new`, so I think I'd like to keep it consistent for now and just use new on these